### PR TITLE
DrawGrammar.0.2.0 and dependency

### PR DESCRIPTION
**Pull request by jacquev6**

### CODE REVIEW

**Comment by jacquev6** [comment]
The Travis build failed because of a transient error from GitHub:

> [ERROR] curl: code 500 while downloading
>        https://github.com/mjambon/easy-format/archive/v1.3.0.tar.gz
> [ERROR] The sources of the following couldn't be obtained, aborting:
>          - easy-format.1.3.0

**Comment by avsm** [comment]
github has been having issues -- thanks for this!

File: packages/General/General.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "_build/src/General.a"
+    "_build/src/General.cmxa"
+    "_build/src/Foundations.cmx"
+    "_build/src/Traits.cmx"
+    "_build/src/Concepts.cmx"
+    "_build/src/Implementation.cmx"
+    "_build/src/Testing.cmx"
+    "_build/src/Specializations.cmx"
+    "_build/src/General.cmx"
+    "%{prefix}%/lib/General"
```
Comment (avsm) [inline packages/General/General.0.2.0/opam]: You may want to consider porting the build to `jbuilder`, which will automate this and also work on byte-code-only architectures.

**Comment by jacquev6** [comment]
Thanks for the merge! I've noticed the excitation around jbuilder these days. I may have a look sometime. Thanks for the tip.

**Pull request merged**